api/HTMLInputElement.json

```
@@ -1286,6 +1286,63 @@
           }
         }
       },
+      "setRangeText": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/HTMLInputElement/setRangeText",
+          "support": {
+            "chrome": {
+              "version_added": true
+            },
+            "chrome_android": {
+              "version_added": true
```

```suggestion
              "version_added": "25"
```
(based on being in Chrome 24)